packages/devtools-reps/src/object-inspector/components/ObjectInspector.css

```diff
@@ -28,17 +28,23 @@
 }
 
 .tree.object-inspector .block .object-label:before {
-  content: "☲ ";
-  font-size: 1.1em;
+  content: "☲";
+  font-size: 1em;
+  display: inline-block;
+  margin-top: -1px;
+  vertical-align: text-top;
```

Comment:
I've had rather different results with text-top/text-bottom depending on the browser/font/operating system.
I'm not sure we can rely on it for precise UI alignment.